docs/testing-guidelines/getting-code-coverage.md

```diff
@@ -0,0 +1,131 @@
+Getting Code Coverage Analysis for PowerShell
+===
+
+**Note: Code coverage is currently only supported on Windows, since we use OpenCover.**
+
+The PowerShell code base is configured to build with code coverage support using [OpenCover](https://github.com/OpenCover/opencover). You can see the testing coverage of the current [`master`](https://github.com/PowerShell/PowerShell)
```

Comment:
Would you bee so kind as to switch to semantic linefeeds and use reference links throughout the document? 

Example:

```markdown
The PowerShell code base is configured to build with code coverage support using [OpenCover].
You can see the testing coverage of the current [`master`] branch build at any time at [codecov.io].

[OpenCover]: https://github.com/OpenCover/opencover
[`master`]: https://github.com/PowerShell/PowerShell
[codecov.io]: https://codecov.io/gh/PowerShell/PowerShell
```

Becomes:

The PowerShell code base is configured to build with code coverage support using [OpenCover].
You can see the testing coverage of the current [`master`] branch build at any time at [codecov.io].

[OpenCover]: https://github.com/OpenCover/opencover
[`master`]: https://github.com/PowerShell/PowerShell
[codecov.io]: https://codecov.io/gh/PowerShell/PowerShell

Makes it easier to track changes.

Reply:
Ah that's a nice feature! Will do.